application/client/src/app/ui/elements/transport/details/serial/styles.less

```diff
@@ -1,8 +1,13 @@
 @import '../../../../styles/variables.less';
 
-
 :host {
     position: relative;
     display: block;
     width: 100%;
-}
\ No newline at end of file
+    & table.settings {
+        border: 0px;
+    }
```

Comment:
on tables browser still reserve some space for border. To get rid of borders (and space) at all, you can use
```
border-collapse: collapse;
```